types/hexo/index.d.ts

```diff
@@ -0,0 +1,976 @@
+// Type definitions for hexo 3.8
+// Project: https://hexo.io/
+// Definitions by: KentarouTakeda <https://github.com/kentarouTakeda>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import EventEmitter = require('events');
+import moment = require('moment');
+import { ParsedArgs } from 'minimist';
+import Logger = require('bunyan');
+import underscore = require('underscore');
+import connect = require('connect');
+import Stream = require('stream');
+import util = require('hexo-util');
+import fs = require('fs');
+
+declare global {
```

Comment:
Does this need to be declared in the global scope?

Reply:
Thank you for your review. I think it is necessary.

Hexo provides a plugin system. Plugin works inside hexo. Then global value `hexo` is provided by hexo.
https://hexo.io/docs/themes#scripts

* TypeScript  
  https://github.com/OpportunityLiu/hexo-renderer-ts/blob/master/index.ts
* JavaScript  
  https://github.com/hexojs/hexo-deployer-git/blob/master/index.js

If there is any other good way, please let me know. I fill follow it.